### PR TITLE
[Python3]move rest cpu test cases as python3 compliant

### PR DESCRIPTION
**Pull request by PraveenPenguin**
move rest cpu test cases as python3 compliant

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: cpu/ebizzy.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def test(self):
                                                       seconds, num_threads)
         args = args + ' ' + args2
 
-        results = process.system_output('%s/ebizzy %s' % (self.sourcedir, args))
+        results = process.system_output('%s/ebizzy %s' % (self.sourcedir, args)).decode("utf-8")
```
Comment (narasimhan-v) [inline cpu/ebizzy.py]: pep8 ?

File: cpu/sensors.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def check_errors(cmd):
         Checks any errors in command result
         """
         errs = []
-        s_output = process.run(cmd, ignore_status=True).stderr
+        s_output = str(process.run(cmd, ignore_status=True).stderr)
```
Comment (narasimhan-v) [inline cpu/sensors.py]: This can be `process.system_output().decode('utf-8')`

File: cpu/cpustress.py [hunk 3]
```diff
@@ -37,7 +37,7 @@
 
 
 def collect_dmesg(object):
-    object.whiteboard = process.system_output("dmesg")
+    object.whiteboard = process.system_output("dmesg").decode()
```
Comment (narasimhan-v) [inline cpu/cpustress.py]: better to specify `utf-8`

File: cpu/sensors.py (hunk 2 above)
Comment (PraveenPenguin) [reply cpu/sensors.py]: @narasimhan-v i tried but that not worked so tried this way

File: cpu/ebizzy.py (hunk 1 above)
Comment (PraveenPenguin) [reply cpu/ebizzy.py]: ok

File: cpu/cpustress.py (hunk 3 above)
Comment (PraveenPenguin) [reply cpu/cpustress.py]: ok

**Comment by PraveenPenguin** [comment]
@narasimhan-v pushed the changes , please have a look

**Comment by PraveenPenguin** [comment]
@harish-24 pushed the changes , please have a look

**Pull request merged**